code/unclassified/src/leap_year/leap_year.js

```diff
@@ -0,0 +1,13 @@
+// Part of cosmos from opengenus foundation
+
+const isLeep = year => ((year % 4 == 0) && (year % 100 != 0)) || (year % 400 == 0)
+
+
```

Comment:
Remove extra newline. Also call it `isLeap`.

Reply:
sorry. typing mistook :laughing: